src/index.html

```diff
@@ -15,8 +15,84 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#logo"
+      >
+        <img
+          class="logo__image"
+          src="images/logo.png"
+          alt="Moyo Company Logo"
+        />
+      </a>
+      <nav>
```

Comment:
It's better to use semantic tags where possible. In this case, you can use `<nav class="nav">` instead of just `<nav>`. It improves your page SEO and helps screen readers.